### ISSUE TITLE
Inspection on the cause of low train loss & high test loss

### ISSUE THREAD

**Issue by dalbom** [KIST-Iceberg/Iceberg#4]
Customizing model
- Dropout rate variation: 0.1 - 0.3
- Testing various linear units: ReLU, ELU

**Comment by dalbom** [comment]
Learning rate를 0.001에서 0.002와 0.0002로 바꿔서 테스트

두 경우 모두 원래의 0.001 보다 좋은 결과를 내진 않음

0.0002보다는 0.002가 나은 결과를 보임

**Comment by dalbom** [comment]
angle이 NaN 인 데이터를 제외하고 돌리면 결과가 확연히 좋아짐.

ELU, PReLU, LeakyReLU 등은 ReLU 보다 좋은 결과를 내지 못함.

Conv와 Activation 사이에 BatchNormalization 레이어를 넣어주면 결과 조금 향상됨.

Cyclic Learning Rate를 적용하면 결과 조금 향상됨.

**Comment by dalbom** [comment]
세번째 채널을 |HH| * |HV| 값으로 할 경우 결과 약간 향상됨

**Comment by kairos03** [comment]
세부 이슈로 분리
